usaspending_api/search/delta_models/award_search.py

```diff
@@ -115,7 +115,7 @@
   awards.type,
   awards.type_description,
   awards.generated_unique_award_id,
-  awards.piid AS display_award_id,
+  COALESCE(awards.piid, awards.fain, awards.uri) AS display_award_id,
```

Comment:
Looks like the pre2008 matview has a case statement to calculate this value:
```
    "  CASE",
    "    WHEN awards.type IN ('02', '03', '04', '05', '06', '10', '07', '08', '09', '11') AND awards.fain IS NOT NULL THEN awards.fain",
    "    WHEN awards.piid IS NOT NULL THEN awards.piid  -- contracts. Did it this way to easily handle IDV contracts",
    "    ELSE awards.uri",
    "  END AS display_award_id,",
```
Is this necessary here instead of the simple coalesce?